tests/functional/spec/features/calculated_summary/new_calculated_summary_repeating_and_static_answers.spec.js

```diff
@@ -173,8 +173,28 @@ describe("Calculated summary with repeating answers", () => {
     await assertSummaryValues(["3", "7"]);
     await $(CalculatedSummaryVisitsPage.submit()).click();
 
-    await expect(await $(Section1SummaryPage.supermarketsListLabel(1)).getText()).to.equal("Lidl");
-    await expect(await $(Section1SummaryPage.supermarketsListLabel(2)).getText()).to.equal("Sainsburys");
-    await expect(await $(Section1SummaryPage.supermarketsListLabel(3)).isExisting()).to.be.false;
+    await expect(await $(SummaryPage.supermarketsListLabel(1)).getText()).to.equal("Lidl");
+    await expect(await $(SummaryPage.supermarketsListLabel(2)).getText()).to.equal("Sainsburys");
+    await expect(await $(SummaryPage.supermarketsListLabel(3)).isExisting()).to.be.false;
+  });
+
+  it("Given I proceed to the second section, the calculated summary of days I visit the supermarket is used as a maximum to the first question", async () => {
+    await $(SummaryPage.submit()).click();
+    await $(HubPage.submit()).click();
+    await $(SupermarketTransportPage.weeklyCarTrips()).setValue(11);
+    await $(SupermarketTransportPage.submit()).click();
+    await expect(await $(SupermarketTransportPage.singleErrorLink()).getText()).to.contain("Enter an answer less than or equal to 10");
+    await $(SupermarketTransportPage.weeklyCarTrips()).setValue(9);
+    await $(SupermarketTransportPage.submit()).click();
+    await expect(await browser.getUrl()).to.contain(SupermarketTransportCostPage.pageName);
+  });
+
+  it("Given I reach the final block, the calculated summary of dynamic answers is piped in correctly", async () => {
+    await $(SupermarketTransportCostPage.weeklyTripsCost()).setValue(30);
+    await $(SupermarketTransportCostPage.submit()).click();
+    await expect(await $("body").getText()).to.have.string("Total weekly supermarket spending: £380.00");
```

Comment:
Maybe we should use this instead? `import CalculatedSummaryPipingPage from "../../../generated_pages/new_calculated_summary_repeating_and_static_answers/calculated-summary-piping.page.js";
`

Reply:
So it is more explicit?

Reply:
I can't see a way to do the assertion with the actual Page, but I've asserted that we're on that page just before to try and make that clearer - if you know another way I'd happily change it